contributing-guides/translation-templates/common-arguments.md

```diff
@@ -11,6 +11,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 | bn    |                      |                        |                                   |           |                   |
 | bs    |                      |                        |                                   |           |                   |
 | ca    | camí/al/fitxer       | camí/al/directori      | camí/al/fitxer_o_directori        | paquet    | nom_usuari        |
+| cs    | cesta/k/souboru      | cesta/k/adresari       | cesta/k/souboru_ci_adresari       | balíček   | username          |
```

Comment:
Kinda curious, Isn't there an equivalent word for username in Czech? Or do you guys prefer using the English word?

_This is what I found on Google translate uživatelské jméno_

Reply:
This is my bad, sorry. I didn't noticed the line wraps up to next screen row, I thought "packet" is the last entry. Will update asap.